modules/nw-dns-forward.adoc

```diff
@@ -5,7 +5,16 @@
 [id="nw-dns-forward_{context}"]
 = Using DNS forwarding
 
-You can use DNS forwarding to override the forwarding configuration identified in `/etc/resolv.conf` on a per-zone basis by specifying which name server should be used for a given zone. If the forwarded zone is the Ingress domain managed by {product-title}, then the upstream name server must be authorized for the domain.
+You can use DNS forwarding to change the forwarding configuration settings in the following ways:
+
+* Override forwarding configuration in the `/etc/resolv.conf` file on a per-zone basis by specifying which name server should be used for a given zone. If the forwarded zone is the Ingress domain managed by {product-title}, then the upstream name server must be authorized for the domain.
+* Override the default upstream in the `/etc/resolv.conf` file by providing a list of upstream DNS servers.
+* Override the default forwarding policy.
```

Comment:
This would benefit from line or two that tells us how this 'override' process works. When I read this, it takes me some time to find the connection between these three methods and the configuration below. I also get confused about the difference between /etc/resolv.conf and the object we modify in the procedure "Sample DNS".

Two other ideas:
1. It might help to link each of the three methods with the exact key or keys that you modify.
2. It might be better to describe the process only in a general way before the procedure. You can add a description of the three methods closer to the sample file.

Reply:
> It might be better to describe the process only in a general way before the procedure. You can add a description of the three methods closer to the sample file.

+1 to this suggestion. I also find it takes some time to understand how the intro correlates to the example.

Reply:
@ctauchen @bobfuru Thanks for the comments. I have made changes. Please check.